Plugins/Public/autobuy/Main.cpp

```diff
@@ -288,19 +315,38 @@ void AutobuyInfo(uint iClientID)
 	PrintUserCmdText(iClientID, L"Error: Invalid parameters");
 	PrintUserCmdText(iClientID, L"Usage: /autobuy <param> [<on/off>]");
 	PrintUserCmdText(iClientID, L"<Param>:");
-	PrintUserCmdText(iClientID, L"   info - display current autobuy-settings");
-	PrintUserCmdText(iClientID, L"   missiles - enable/disable autobuy for missiles");
-	PrintUserCmdText(iClientID, L"   torps - enable/disable autobuy for torpedos");
-	PrintUserCmdText(iClientID, L"   mines - enable/disable autobuy for mines");
-	PrintUserCmdText(iClientID, L"   cd - enable/disable autobuy for cruise disruptors");
-	PrintUserCmdText(iClientID, L"   cm - enable/disable autobuy for countermeasures");
-	PrintUserCmdText(iClientID, L"   reload - enable/disable autobuy for nanobots/shield batteries");
-	PrintUserCmdText(iClientID, L"   repair - enable/disable auto-repair");
-	PrintUserCmdText(iClientID, L"   all: enable/disable autobuy for all of the above");
+	PrintUserCmdText(iClientID, L"|   info - display current autobuy-settings");
+	PrintUserCmdText(iClientID, L"|   missiles - enable/disable autobuy for missiles");
+	PrintUserCmdText(iClientID, L"|   torps - enable/disable autobuy for torpedos");
+	PrintUserCmdText(iClientID, L"|   mines - enable/disable autobuy for mines");
+	PrintUserCmdText(iClientID, L"|   cd - enable/disable autobuy for cruise disruptors");
+	PrintUserCmdText(iClientID, L"|   cm - enable/disable autobuy for countermeasures");
+	PrintUserCmdText(iClientID, L"|   bb - enable/disable autobuy for nanobots/shield batteries");
+	PrintUserCmdText(iClientID, L"|   munition - enable/disable autobuy for ammo");
+	PrintUserCmdText(iClientID, L"|   cloak - enable/disable autobuy for cloak batteries");
```

Comment:
uh.... yes. I see
those things under extra
[extra]
item = dsy_cloak_1, missile01_mark01_ammo
item = dsy_cloak_2, missile01_mark01_ammo
item = dsy_cloak_3, missile01_mark01_ammo
item = dsy_cloak_4, missile01_mark02_ammo
item = dsy_cloak_1_n, missile01_mark04_ammo
item = dsy_cloak_2_n, missile01_mark04_ammo
item = dsy_cloakdisruptor_1, missile01_mark01_ammo
item = dsy_cloakdisruptor_2, missile01_mark01_ammo
item = dsy_cloakdisruptor_3, missile01_mark01_ammo

as 
PrintUserCmdText(iClientID, L"|   cloak - enable/disable autobuy for cloak batteries");
Since they are both using cloak batteries

Reply:
yes, also:
```
item = dsy_hspace_jump_drive_1, mine01_mark01_ammo
item = dsy_hspace_jump_drive_2, mine01_mark01_ammo
item = dsy_hspace_jump_drive_3, mine01_mark01_ammo 
```

should we end support for `extra` and introduce 3 new categories from config in its place, having cloak only be the cloaks?

Reply:
That what I was thinking and made basically
Extra was only remaining to be named as extra... in reality it is just cloak batteries for cloaks and cloak disruptors